.gitignore

```diff
@@ -234,23 +233,33 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
-/.vs
 
 ##########
 # Rider
 ##########
 .idea/
 
+##########
+# Visual Studio Code
+##########
+.vscode/
```

Comment:
This is already in the repo, @anthonylavado

Reply:
I don't remember commenting about it, but it's good that it's there :sweat_smile: